### DISCUSSION TITLE
Use with pre-inited GTM

### DISCUSSION THREAD

**Discussion by mikaraunio** [gtm-support/vue-gtm#474] [Q&A] [answered]
Is there a straightforward way to bypass the GTM script tag injection, so this could be used in Vue components in hybrid web apps where the script tag has already been rendered server-side?

Of course we can just use `dataLayer.push()` directly in that scenario, but it would be nice to get the router integration, and a consistent interface across projects.

**Comment by Shinigami92** [comment, marked as answer]
Maybe explore the code a bit around here: https://github.com/gtm-support/core/blob/1ec51bc62fd192d49f18ad7a65535d19998a1705/src/gtm-support.ts#L96-L101

I think you could make use the of `enable` function.

**Reply by mikaraunio** [reply]
Yes, thanks! I can just set the `loadScript` option to false. It's right there in the docs, don't know how I missed it.
